pkg/gadgets/trace/fsslower/tracer/gadget.go

```diff
@@ -63,7 +63,7 @@ func (g *GadgetDesc) ParamDescs() params.ParamDescs {
 			Title:          "Filesystem",
 			DefaultValue:   "ext4",
 			Description:    "Filesystem to trace",
-			PossibleValues: []string{"btrfs", "ext4", "nfs", "xfs"},
+			PossibleValues: []string{"btrfs", "cifs-direct", "cifs-loose", "cifs-strict", "ext4", "nfs", "xfs"},
```

Comment:
This only covers parts of the possible mount options: it does not handle the CIFS_MOUNT_NO_BRL option (`-o nobrl` is orthogonal to the `-o cache=` options.